jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -635,7 +644,7 @@ def user_api(self):
         expose = ['multiselect', 'viewer', 'viewer_multiselect', 'layer', 'layer_multiselect',
                   'select_all', 'subset_visible', 'reset_viewer_bounds']
         if self.config == "cubeviz":
-            expose += ['uncertainty_visible']
+            expose += ['uncertainty_visible', 'volume_level', 'sonification_enable']

```

Comment:
these should also be in the docstring for the class.  Do we want to call this `sonfication_enabled`/`audio_enabled` (I think we use enabled elsewhere instead of enable since that sounds like a verb), or just `enabled` or maybe even `audible`?  Maybe @Jenneh has thoughts, or if we're not ready to commit, we can leave out of the public API for now.

Reply:
I changed `sonification_enable` to `sonification_enabled` and added that and `volume_level` to the docstrings for Plot Options.

Reply:
Sounds good, changed in the latest commit.